server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -95,120 +100,97 @@
 
     private final boolean trackRealMemoryUsage;
     private volatile BreakerSettings parentSettings;
-    private volatile BreakerSettings fielddataSettings;
-    private volatile BreakerSettings inFlightRequestsSettings;
-    private volatile BreakerSettings requestSettings;
-    private volatile BreakerSettings accountingSettings;
 
     // Tripped count for when redistribution was attempted but wasn't successful
     private final AtomicLong parentTripCount = new AtomicLong(0);
 
     public HierarchyCircuitBreakerService(Settings settings, ClusterSettings clusterSettings) {
```

Comment:
Why do we need this alternate ctor? Can we convert the few tests calling tihs to pass an empty list?